src/supervisor.lisp

```diff
@@ -29,7 +29,13 @@
     :accessor supervisor-node-class
     :initform 'blossom-node
     :initarg :node-class
-    :type symbol))
+    :type symbol)
+   (node-dryad
+    :accessor supervisor-node-dryad
+    :initform nil
```

Comment:
It looks like this field is always set. If that's the case, you don't _need_ an `:initform`. It can also be useful to have unbound slot accesses throw errors rather than quietly returning `NIL` (which probably then someone else will throw an error on — here some poor `send-message` caller).